framework/src/kernels/Kernel.C

```diff
@@ -43,7 +43,9 @@ Kernel::Kernel(const InputParameters & parameters)
     _u(_is_implicit ? _var.sln() : _var.slnOld()),
     _grad_u(_is_implicit ? _var.gradSln() : _var.gradSlnOld()),
     _u_dot(_var.uDot()),
-    _du_dot_du(_var.duDotDu())
+    _u_dotdot(_var.uDotdot()),
```

Comment:
Same question: should these be in Kernel or TimeKernel?

Reply:
These shouldn't be here at all.  Both `_u_dot` and `_u_dotdot` should be guarded by `_need_*` and only computed if they're requested.